src/assets/wise5/services/branchService.ts

```diff
@@ -5,6 +5,9 @@ import { PathService } from './pathService';
 
 @Injectable()
 export class BranchService {
+  readonly CHOICE_CHOSEN: string = 'choiceChosen';
+  readonly SCORE: string = 'score';
```

Comment:
Should these variables be in a separate file, outside of BranchService? Having to depend on BranchService just to use these variables like we do now in BranchPathAuthoringComponent doesn't seem ideal.

Reply:
Good idea, I've made the changes.